compiler/hash-typecheck/src/ops/simplify.rs

```diff
@@ -1399,9 +1460,9 @@ mod test_super {
         (global_storage, local_storage, core_defs, source_map)
     }
 
-    #[test]
-    fn test_simplify() {
-        let (mut global_storage, mut local_storage, core_defs, source_map) = get_storages();
+    // #[test]
+    fn _test_simplify() {
```

Comment:
broken test?

Reply:
I should probably just delete it since it's not even a proper test

Reply:
Sounds good, maybe general tc tests are the way to go